extensions/roc-plugin-dredd/src/config/roc.config.js

```diff
@@ -1 +1,16 @@
-export default {};
+export default {
+    settings: {
+        test: {
+            dredd: {
```

Comment:
Some of the settings would make sense to be able to define on a "global" perspective. Thinking about `level` and `silent`, maybe they eventually can be something that can be defined on the highest level and then used throughout the system. A version of this, although not as detailed, is to change the `level` based on if Roc is started in `verbose` mode or not.